tests/functional/sse-kms-migration/arnPrefix.js

```diff
@@ -602,12 +602,12 @@ describe('KMS error', () => {
         },
         {
             action: 'getObject', kmsAction: 'Decrypt',
-            fct: async () => helpers.s3.getObject({ Bucket, Key }).promise(),
+            fct: async () => helpers.s3.getObject({ Bucket, Key }) ,
         },
         {
             action: 'copyObject', detail: ' when getting from source', kmsAction: 'Decrypt',
             fct: async () =>
-                helpers.s3.copyObject({ Bucket, Key: 'copy', CopySource: `${Bucket}/${Key}` }).promise(),
+                helpers.s3.copyObject({ Bucket, Key: 'copy', CopySource: `${Bucket}/${Key}` }) ,
```

Comment:
```suggestion
                helpers.s3.copyObject({ Bucket, Key: 'copy', CopySource: `${Bucket}/${Key}` }),
```

You have some other iterations of that in your file